src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -62,6 +65,43 @@ class ServiceTypeAhead extends Component {
     return false;
   };
 
+  renderSearchForAvailableServicePrompt = inputValue => {
+    const { isFocused } = this.state;
+    const { showError } = this.props;
+
+    if (
+      (isFocused && !inputValue && !showError) ||
+      (inputValue && inputValue.length < 2)
```

Comment:
Suggestion: define a local constant, e.g. `MIN_SEARCH_CHARS`, and use it here and also on line 167 in place of the hard-coded number "2". Makes the code more readable and makes it easier to change later.

Reply:
sounds good to me